src/app/ui/swap/utils.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link as MuiLink } from '@mui/material';
+import Link from 'next/link';
+import type { ExtendedChain, Token } from '@lifi/sdk';
+import { getChainById } from '@/utils/tokenAndChain';
+import { currencyFormatter } from '@/utils/formatNumbers';
+
+export function buildExplorerLink(
+  blockExplorerUrls: string[] = [],
+  address: string,
+) {
+  if (blockExplorerUrls.length === 0) {
+    return address;
+  }
+
+  return (
+    <MuiLink
+      color="text.primary"
+      component={Link}
+      target="_blank"
+      href={`${blockExplorerUrls[0]}/tokens/${address}`}
+    >
+      {address}
+    </MuiLink>
+  );
+}
+
+export function getChainInfoData(chainInfo: ExtendedChain) {
```

Comment:
can't we reuse the function from the "/bridge" page ?

Reply:
Yes, definitely. It´s now removed and is importing the file from the bridge utils.